xonsh/built_ins.py

```diff
@@ -569,9 +569,9 @@ def run_subproc(cmds, captured=False):
                     e = 'xonsh: subprocess mode: permission denied: {0}'
                     raise XonshError(e.format(cmd[0]))
         _stdin_file = None
-        if (captured == 'object' and
-                stdin is not None and
-                __xonsh_env__['XONSH_STORE_STDIN'] and
+        if (stdin is not None and
+                ENV.get('XONSH_STORE_STDIN') and
```

Comment:
I don't really understand the insistence to make this a `get` call since it won't change the behavior at all (unless someone removes `XONSH_STORE_STDIN` from the _defaults_ dictionary).  But it doesn't bother me at all.

Is there something I'm missing?